.htaccess

```diff
@@ -0,0 +1 @@
+ErrorDocument 404 /404.html
```

Comment:
Github doesn't use Apache for hosting, and already handles 404 pages-

https://docs.github.com/en/github/working-with-github-pages/creating-a-custom-404-page-for-your-github-pages-site

Reply:
removed htaccess file, it should work without it fine